src/mgr/StandbyPyModules.h

```diff
@@ -81,9 +81,10 @@ class StandbyPyModule : public PyModuleRunner
   StandbyPyModule(
       StandbyPyModuleState &state_,
       const PyModuleRef &py_module_,
-      LogChannelRef clog_)
+      LogChannelRef clog_,
+      LogChannelRef audit_clog_)
     :
-      PyModuleRunner(py_module_, clog_),
+      PyModuleRunner(py_module_, clog_, audit_clog_),
```

Comment:
@jcsp I assume that we can remove this changes after dropping StandbyMgrModule::audit_log() in mgr_module.py?